pkg/build/iso.go

```diff
@@ -146,3 +153,27 @@ func (b *Builder) createIsoCommand(logFilename, scriptName string) (*exec.Cmd, *
 
 	return cmd, logFile, nil
 }
+
+func (b *Builder) findExtractedRawImage() (string, error) {
+	var foundFile string
+	f := func(s string, info os.FileInfo, e error) error {
+		if e != nil {
+			return e
+		}
+		if strings.HasSuffix(info.Name(), ".raw") {
+			foundFile = info.Name()
+		}
+		return nil
+	}
+
+	rawExtractPath := filepath.Join(b.context.BuildDir, rawExtractDir)
+	if err := filepath.Walk(rawExtractPath, f); err != nil {
+		return "", fmt.Errorf("traversing raw extract directory %s: %w", rawExtractPath, err)
+	}
+
+	if foundFile == "" {
+		return "", fmt.Errorf("unable to find a raw image in: %s", rawExtractDir)
```

Comment:
Can we add a test case for this?

Reply:
Yeah, that's my bad. I initially thought I'd have to come back when I got around to building a test ISO, but I can test this without the need for that.